Add celery and gravity files to .gitignore

davebx opened the pull request:
## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

davebx committed:
Add celery and gravity files to .gitignore

nsoranzo commented on .gitignore:
```diff
@@ -1,4 +1,10 @@
+# Celery/Gravity state files
+
+celerybeat-schedule.*
+database/gravity
```
Should we just replace the over 20 `database/something` entries with just `database/*` ?

jdavcs replied:
Maybe just add (exclude) a README with a comment (like the current info.txt) and ignore the rest?

dannon replied:
IMO just ignore the whole thing.  If someone needs to change the info or add more tracked files, that's always still possible.

dannon replied:
I pushed a commit doing this; hope that's fine @davebx

dannon committed:
Ignore all database contents by default; group celery ignores more
locally.

Prefix /database/ so we only ignore the one at root.  (wouldn't be
surprised if we had another dir named 'database' somewhere)
